### PR TITLE
generate python package

### PR DESCRIPTION
**Pull request by pnuckowski**
alamo-alerter package created based on cookiecutter :)

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[tox]
+envlist = py26, py27, py33, py34
+
+[testenv]
+setenv =
+    PYTHONPATH = {toxinidir}:{toxinidir}/alamo_alerter
+commands = python setup.py test
+deps =
+    -r{toxinidir}/requirements.txt
```
Comment (deejay1) [inline tox.ini]: drop py26 and 33, add flake checks?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,81 @@
+.PHONY: clean-pyc clean-build docs clean
```
Comment (deejay1) [inline Makefile]: do we really need this files

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Config file for automatic testing at travis-ci.org
+
+language: python
+
+python:
+  - "3.4"
+  - "3.3"
+  - "2.7"
+  - "2.6"
+  - "pypy"
```
Comment (deejay1) [inline .travis.yml]: we can drop 2.6 and 3.3 here also

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+# command to install dependencies, e.g. pip install -r requirements.txt --use-mirrors
+install: pip install -r requirements.txt
+
+# command to run tests, e.g. python setup.py test
+script: python setup.py test
```
Comment (deejay1) [inline .travis.yml]: also maybe implement tox support from https://github.com/RulersOfAsgard/ALAMO-scheduler/blob/develop/.travis.yml

**Comment by benethor** [comment]
:+1:

File: setup.py [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+        "Programming Language :: Python :: 2",
+        'Programming Language :: Python :: 2.6',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.3',
+        'Programming Language :: Python :: 3.4',
+    ],
+    test_suite='tests',
+    tests_require=test_requirements
+)
```
Comment (deejay1) [inline setup.py]: again - pbr and setup.cfg ?

Comment (pnuckowski) [inline setup.py]: What is wrong with this?

File: tox.ini [hunk 6]
```diff
@@ -0,0 +1,42 @@
+[tox]
+envlist = py27, py34, pep8, coverage
```
Comment (ch40s3k) [inline tox.ini]: remove py27

File: setup.py (hunk 5 above)
Comment (ch40s3k) [inline setup.py]: We are using pbr: http://docs.openstack.org/developer/pbr/ . There is nothing wrong with this, but it is better to keep one convention.

Comment (pnuckowski) [inline setup.py]: +1 fixed :)

File: AUTHORS [hunk 7]
```diff
@@ -0,0 +1 @@
+Pawel Nuckowski <pawel.nuckowski@allegrogroup.com>
```
Comment (deejay1) [inline AUTHORS]: AFAIR authors is being generated by pbr

File: ChangeLog [hunk 8]
```diff
@@ -0,0 +1,5 @@
+CHANGES
+=======
+
+* generate python package
+* initial commit
```
Comment (deejay1) [inline ChangeLog]: PBR also does changelog

File: MANIFEST.in [hunk 9]
```diff
@@ -0,0 +1,11 @@
+include AUTHORS.rst
```
Comment (deejay1) [inline MANIFEST.in]: AUTHORS sans the .rst

File: HISTORY.rst [hunk 10]
```diff
@@ -0,0 +1,9 @@
+.. :changelog:
```
Comment (deejay1) [inline HISTORY.rst]: really? drop it please and remove from MANIFEST.in

File: MANIFEST.in (hunk 9 above)
Comment (deejay1) [inline MANIFEST.in]: also add changelog from pbr here

File: AUTHORS (hunk 7 above)
Comment (pnuckowski) [inline AUTHORS]: Yes it is. This is a result of pbr....

File: ChangeLog (hunk 8 above)
Comment (pnuckowski) [inline ChangeLog]: Generated by pbr

**Comment by deejay1** [comment]
:+1:

**Comment by deejay1** [comment]
hipchat integration test ;)

**Comment by benethor** [comment]
:+1:

**Pull request merged**